Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM fedora:28
+FROM centos:7
 
 MAINTAINER "Karel Šimon" <ksimon@redhat.com>
 
 ENV container docker
 
+RUN yum -y update
+RUN yum clean all
+
```

Comment:
(Not really related to Dockerfile but worth mentioning early)
It seems a good idea to have a dependency management tool (gopkg.toml, glide.yaml) and to add the vendored deps to the repo

Reply:
for now, this plugin doesn't have any vendor dependency. But sure if anything shows, i will add go dep.